duckduckgo/Android
# Revert continuation indent to be 4

Task/Issue URL: https://app.asana.com/0/414730916066338/1201492317527942/f

### Description
In [this](https://app.asana.com/0/414730916066338/1201355216612743) task we make some improvements but changed some of the IDE configs we have before, like the continuation indent that went from 4 -> 8

This PR reverts that value to be 4

### Steps to test this PR
1. pull this branch
1. Open the IDE
1. verify that continuation indents are not massive anymore
